src/extension.ts

```diff
@@ -122,9 +123,17 @@ function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, c
 				result = 'Canceled';
 			}
 			else if (err instanceof Error) {
-				vscode.window.showErrorMessage(err.message);
+				output.appendLine(err.message);
+				if (JSON.parse(err.message).message.includes("\n") || JSON.parse(err.message).message.includes("\r")) {
+					output.show();
+					vscode.window.showErrorMessage('An error has occured. See output window for more details.');
+				}
+				else {
+					vscode.window.showErrorMessage(err.message);
+				}
 			}
 			else if (typeof err === "string") {
+				output.appendLine(err);
```

Comment:
This could be multiline as well

Reply:
That's why we have the `ErrorData` class in the other repos (instead of 'util.errToString()') so that you can use the ErrorData.message property